### PR TITLE
Tornadoplots, and further improvements to `VolumetricAnalysis`

### PR DESCRIPTION
**Pull request by tnatt**
Various improvement to the `VolumetricAnalysis `plugin.

- [ ] Added page with tornadoplots for sensitivity runs
- [ ] Added function in `InplaceVolumesModel `to retrieve dataframe. The function handles filtering, grouping of the data and addition of parameters. This removed a lot of logic from the callbacks in the `VolumetricAnalysis `plugin.
- [ ] Possible computation of NTG from facies if `non_net _facies` are set in the config 
- [ ] Automatic computation of water zone volumes if total volumes are included
- [ ] Improved callback logic by separating selections/filters by tab.
- [ ] Added "bar" as plot type option
- [ ] Raise exception if selectors are defined for all sources. 
- [ ] Changed to the new layout components in webviz-core-components

---

### Contributor checklist

- [ ] :tada: This PR closes #ISSUE_NUMBER.
- [ ] :scroll: I have broken down my PR into the following tasks:
   - [ ] Task 1
   - [ ] Task 2
- [ ] :robot: I have added tests, or extended existing tests, to cover any new features or bugs fixed in this PR.
- [ ] :book: I have considered adding a new entry in `CHANGELOG.md`, and added it if should be communicated there.

### CODE REVIEW

File: webviz_subsurface/_models/inplace_volumes_model.py [hunk 1]
```diff
@@ -32,74 +30,94 @@ class InplaceVolumesModel:
...
-                "Ensembles with and without sensitivity data mixed - this is not supported \n"
-                f"Sensitivity ensembles: {df.loc[~df['SENSNAME'].isnull()]['ENSEMBLE'].unique()} "
-                f"Non-sensitivity ensembles: {df.loc[df['SENSNAME'].isnull()]['ENSEMBLE'].unique()}"
-            )
+        # compute and set property columns
+        self._set_initial_property_columns()
+        self._dataframe = self.compute_property_columns(self._dataframe)
 
-        self.set_initial_property_columns()
+    # ---------------------------------------------------------------------------------
```
Comment (HansKallekleiv) [inline webviz_subsurface/_models/inplace_volumes_model.py:120]: remove

**Review by HansKallekleiv — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [UNRELEASED] - YYYY-MM-DD
 
 ### Changed
+- [#681](https://github.com/equinor/webviz-subsurface/pull/681) - `VolumetricAnalysis` upgrades - added page with tornadoplots to `VolumetricAnalysis`, automatic computation of volumes 
+from the water zone if totals volume are included, and possibility of computing NTG from facies.
```
Comment (anders-kiaer) [inline CHANGELOG.md:11]: "totals volume"?

File: webviz_subsurface/_models/inplace_volumes_model.py [hunk 3]
```diff
@@ -149,17 +171,46 @@ def responses(self) -> List[str]:
...
+            volcols = [col for col in df if col not in selectors]
+            if not any(col in volcols for col in supported_columns):
+                continue
+            if df[volcols].isnull().values.any():
+                warnings.warn(
+                    f"WARNING: Cannot calculate water zone volumes for source {src}, "
+                    "due to wrong format in input volume file. \nTo ensure correct format "
+                    "use: https://equinor.github.io/fmu-tools/fmu.tools.rms.html#fmu.tools."
+                    "rms.volumetrics.merge_rms_volumetrics"
+                )
```
Comment (anders-kiaer) [inline webviz_subsurface/_models/inplace_volumes_model.py:192]: Nice!

File: webviz_subsurface/_models/inplace_volumes_model.py [hunk 4]
```diff
@@ -168,33 +219,110 @@ def set_initial_property_columns(self) -> None:
...
+
+        self._parameterdf["SENSTYPE"] = self._parameterdf.apply(
+            lambda row: find_sens_type(row.SENSCASE)
+            if not pd.isnull(row.SENSCASE)
+            else np.nan,
+            axis=1,
+        )
+        sens_params_table = self._parameterdf[self.SENS_COLUMNS]
+
+        # TODO do this per ensemble?
```
Comment (anders-kiaer) [inline webviz_subsurface/_models/inplace_volumes_model.py:234]: Do we have an issue on this? If not create one perhaps in order to remember :slightly_smiling_face:

**Review by anders-kiaer — CHANGES_REQUESTED** [review]
LGTM! :100: Only minor suggestions.

**Review by anders-kiaer — APPROVED** [review]

**Pull request merged**